Functions/xASL_quant_Basil.m

```diff
@@ -0,0 +1,411 @@
+function [ScaleImage, CBF] = xASL_quant_Basil(PWI, M0_im, SliceGradient, x)
```

Comment:
Yes indeed - much of the latter part of the function are shared with the existing quantification. My view was that as I don't have much familiarity with the code structure generally (and since Matlab isn't my main are of knowledge!) that this potential refactoring would be better done by the exploreASL team. If I did it I'd be working a bit blind and there's a high chance I might mess things up.